web/context.go

```diff
@@ -743,7 +743,7 @@ func (c *Context) RequireInvoiceId() *Context {
 		return c
 	}
 
-	if len(c.Params.InvoiceId) != 27 {
+	if len(c.Params.InvoiceId) != 27 && c.Params.InvoiceId != model.UpcomingInvoice {
```

Comment:
What are we testing here checking against `upcoming` in the `id` for get invoice requests?

Reply:
Nvm, saw your webapp PR now that requests the invoice with `upcoming` as Id